core/apiCalls.js

```diff
@@ -108,18 +108,7 @@ export function commitToWorkspaceApi(blueprint) {
 }
 
 export function fetchDiffWorkspaceApi(blueprintId) {
-  const p = new Promise((resolve, reject) => {
-    utils
-      .apiFetch("/api/v0/blueprints/diff/" + blueprintId + "/NEWEST/WORKSPACE")
-      .then(data => {
-        resolve(data);
-      })
-      .catch(e => {
-        console.log("Error fetching diff", e);
-        reject();
-      });
-  });
-  return p;
```

Comment:
Just for clarity, here they were calling reject inside the catch, so there was not an issue of not propagating the exception up. But I agree that it's useless and well removed.